### ISSUE TITLE
UpdateOrders param default orderset = NULL

### ISSUE THREAD

**Issue by edward-wilson** [braverock/quantstrat#50]
I am calling `updateOrders` with the `orderset` param left as default which is `NULL`. Line 524 of `orders.R` in `updateOrders` is calling `is.na()` on the `orderset` param which is causing an `argument is of length zero` error. 

Specifically in my case, I am calling `updateOrders` from the `ruleRevoke` function. If I set `orderset = NA` then I no longer get the error.

**Comment by joshuaulrich** [comment]
This was also reported by someone else via email. Their suggestion was `if(!is.null(orderset) && is.na(orderset)) orderset=NULL`. Here's my response to the email:

> It looks like that line is in R/orders.R:L524 because ruleSignal sets orderset=NA if it's NULL.
> https://github.com/braverock/quantstrat/blob/master/R/ruleSignal.R#L201
>
> So your suggestion, or adding the following to R/ruleRevoke.R:L38 would probably work:
> if(is.null(orderset)) orderset <- NA
>
> That said, I'd prefer a solution that avoided NA, NULL, etc entirely. Maybe by setting orderset="" by default?

**Comment by edward-wilson** [comment]
I think the solution should be in the `updateOrders` function itself, in case users call the function directly from a custom rule function for example. 

My proposed solution would be: `if(!is.null(orderset) || is.na(orderset)) orderset=NULL` on L524 in `orders.R`.

**Comment by joshuaulrich** [comment]
Your proposed solution is wrong. It would set `orderset = NULL` if `orderset` was not null. You probably meant to agree with the proposed solution by the reporter who responded via email.

**Comment by edward-wilson** [comment]
Yes apologies. I realised as soon as I commented. Ive edited previous comment to suit.
